adagrams/game.py

```diff
@@ -1,11 +1,107 @@
+
+# import random
+import random
+
+import copy 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+letter_list = []
+for letter in LETTER_POOL.keys():
+    letter_list += LETTER_POOL[letter] * letter
+# print(letter_list)
+
 def draw_letters():
-    pass
+    selected_letters = []
+    while len(selected_letters) < 10:
+        random_index = random.randint(0,  len(letter_list)-1)
+        chosen_letter = letter_list[random_index]
+        if selected_letters.count(chosen_letter) < LETTER_POOL[chosen_letter]:
+            selected_letters += chosen_letter
+            # print(selected_letters)
+            
+        else:
+            continue
+    print(selected_letters)
+    return selected_letters
+      
+      
+
```

Comment:
It’s good practice to avoid committing extra or white space when submitting a PR.